site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -620,7 +646,83 @@ components:
          description: Allows arbitrary properties as extension to the ones
                       specified by the Hono API.
 
-# Tenant schema
+
+      PagingSortingFilteringOptions:
+         type: object
+         additionalProperties: false
+         properties:
+            "page":
+               $ref: '#/components/schemas/PagingOptions'
+            "filters":
+               type: array
+               description: A list of filter expressions. Objects must match all filters in order to be included in a result.
+               items:
+                  $ref: '#/components/schemas/FilterOption'
+            "sort":
+               type: array
+               description: A list of sorting options. If multiples options are provided, the results will be sorted following the order of the options in the list.
+               items:
+                  $ref: '#/components/schemas/SortingOption'
+
+      PagingOptions:
+         type: object
+         additionalProperties: false
+         description: options to page the results of bulk queries.
+         properties:
+            "limit":
+               type: integer
+               default: 30
+               minimum: 0
+               maximum: 200
+               description: The maximum number of objects to include in a response.
+            "offset":
+               type: integer
+               default: 0
+               minimum: 0
+               description: |
+                  The offset to use as a starting point to return results. This allows to go through the query results in
+                  a paged manner when the query returns more results than the limit.
+
+      FilterOption:
+            type: object
+            additionalProperties: false
+            description: Filters to be applied to bulk queries.
+            properties:
+               "field":
+                  type: string
+                  description: The JSON Path representation of the field to use for filtering.
+               "value":
+                  type: string
+                  description: |
+                     The value to filter the field with.
+                     SQL wildcard characters are supported : `%` will match zero or any number of characters
+                     and `_` will match exactly one character.
```

Comment:
FMPOV standard wildcards like `*` (zero or more characters) and `?` one character would be more appropriate as they are more widely known.
Just out of curiosity: if *eq* allows for using wildcards, what are be semantics of the *like* operation? I sthat about phonetic similarity or similar spelling? 
Based on the fact that we are allowing wildcards, the operation that we are talking about would rather be *like* and not *eq*, right?

Reply:
I went with `%` and `_` because they are standard SQL wildcards, where `*` and `?` are more used in shell environment. I have no preference there. 

And yeah, I agree, I would intuitively say that `eq *word*` is similar to `like word`, but it probably changes with actual database implementations

Reply:
Let's do a poll . 
@dejanb  @ctron @sophokles73 @calohmn  please vote on that comment :
:+1: for `*` and `?`
:-1: for `%` and `_`

Reply:
I see that REST APIs use mostly */? combination, so let's go with that.

Reply:
I updated the PR with */? as wildcards.

Reply:
did you?

Reply:
ahem, sorry.. This should be fixed :grin: